rollup.config.js

```diff
@@ -3,31 +3,87 @@ import { terser } from "rollup-plugin-terser";
 
 export default [
   {
-    input: "src/single-spa-react.js",
+    input: `src/single-spa-react.js`,
+    output: [
+      {
+        file: `lib/umd/single-spa-react.js`,
+        format: "umd",
+        name: "singleSpaReact",
+        sourcemap: true,
+      },
+      {
+        file: `lib/system/single-spa-react.js`,
+        format: "system",
+        sourcemap: true,
+      },
+      {
+        file: `lib/esm/single-spa-react.js`,
+        format: "esm",
+        sourcemap: true,
+      },
+    ],
+    plugins: [babel(), terser()],
+  },
+  {
+    input: "./src/single-spa-react.js",
     output: {
-      file: "lib/single-spa-react.js",
-      format: "umd",
-      name: "singleSpaReact",
+      file: `./lib/es2015/single-spa-react.js`,
+      format: "esm",
       sourcemap: true,
     },
+    plugins: [
+      babel({
```

Comment:
Let's upgrade to `@rollup/plugin-babel` instead of `rollup-plugin-babel` (since that is deprecated). See https://github.com/rollup/plugins/tree/master/packages/babel

Reply:
Ok